code/modules/clothing/neck/scarfs.dm

```diff
@@ -0,0 +1,84 @@
+/obj/item/clothing/neck/scarf
+	name = "scarf"
+	desc = "A stylish scarf. The perfect winter accessory for those with a keen fashion sense, and those who just can't handle a cold breeze on their necks."
+	w_class = WEIGHT_CLASS_SMALL
+	dog_fashion = /datum/dog_fashion/head
+
+/obj/item/clothing/neck/scarf/red
+	name = "red scarf"
+	icon_state = "redscarf"
+	item_color = "redscarf"
+
+/obj/item/clothing/neck/scarf/green
+	name = "green scarf"
+	icon_state = "greenscarf"
+	item_color = "greenscarf"
+
+/obj/item/clothing/neck/scarf/darkblue
+	name = "dark blue scarf"
+	icon_state = "darkbluescarf"
+	item_color = "darkbluescarf"
+
+/obj/item/clothing/neck/scarf/purple
+	name = "purple scarf"
+	icon_state = "purplescarf"
+	item_color = "purplescarf"
+
+/obj/item/clothing/neck/scarf/yellow
+	name = "yellow scarf"
+	icon_state = "yellowscarf"
+	item_color = "yellowscarf"
+
+/obj/item/clothing/neck/scarf/orange
+	name = "orange scarf"
+	icon_state = "orangescarf"
+	item_color = "orangescarf"
+
+/obj/item/clothing/neck/scarf/lightblue
+	name = "light blue scarf"
+	icon_state = "lightbluescarf"
+	item_color = "lightbluescarf"
+
+/obj/item/clothing/neck/scarf/white
+	name = "white scarf"
+	icon_state = "whitescarf"
+	item_color = "whitescarf"
+
+/obj/item/clothing/neck/scarf/black
+	name = "black scarf"
+	icon_state = "blackscarf"
+	item_color = "blackscarf"
+
+/obj/item/clothing/neck/scarf/zebra
+	name = "zebra scarf"
+	icon_state = "zebrascarf"
+	item_color = "zebrascarf"
+
+/obj/item/clothing/neck/scarf/christmas
+	name = "christmas scarf"
+	icon_state = "christmasscarf"
+	item_color = "christmasscarf"
+
+//The three following scarves don't have the scarf subtype
+//This is because Ian can equip anything from that subtype
+//However, these 3 don't have corgi versions of their sprites
```

Comment:
Couldn't you just set their dog _fashions to null?

Reply:
Yup, that works.